nemo_skills/prompt/config/deepseek/minif2f.yaml

```diff
@@ -12,21 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-executor: local
+few_shot_examples:
+  prefix: ""
+  template: |-
+    Complete the following Lean 4 code:
+      ‘‘‘lean4
```

Comment:
Use the right backticks

Reply:
I think in the paper they use exactly this backticks
```
Complete the following Lean 4 code:
‘‘‘lean4
import Mathlib
import Aesop
set_option maxHeartbeats 0
open BigOperators Real Nat Topology Rat
/-- Show that 9𝑥
2
sin2
𝑥+4
𝑥 sin 𝑥
⩾ 12 for 0 < 𝑥 < 𝜋.-/
theorem aime_1983_p9 (x : R) (h0 : 0 < x ∧ x < Real.pi) :
12 ⩽ (9 * (x ^ 2 * Real.sin x ^ 2) + 4) / (x * Real.sin x) := by
```